common/src/main/java/org/opensearch/ml/common/CommonValue.java

```diff
@@ -29,132 +26,134 @@ public class CommonValue {
+        + "\" : {\"type\": \"long\"},\n"
+        + "      \""
+        + MLModel.MODEL_CONFIG_FIELD
+        + "\" : {\"properties\":{\""
+        + MODEL_TYPE_FIELD + "\":{\"type\":\"keyword\"},\""
+        + EMBEDDING_DIMENSION_FIELD + "\":{\"type\":\"integer\"},\""
+        + FRAMEWORK_TYPE_FIELD + "\":{\"type\":\"keyword\"},\""
+        + ALL_CONFIG_FIELD + "\":{\"type\":\"text\"}}},\n"
+        + "      \""
+        + MLModel.MODEL_CONTENT_HASH_VALUE_FIELD
+        + "\" : {\"type\": \"keyword\"},\n"
+        + "      \""
+        + MLModel.CREATED_TIME_FIELD
+        + "\": {\"type\": \"date\", \"format\": \"strict_date_time||epoch_millis\"},\n"
+        + "      \""
+        + MLModel.LAST_UPLOADED_TIME_FIELD
+        + "\": {\"type\": \"date\", \"format\": \"strict_date_time||epoch_millis\"},\n"
+        + "      \""
+        + MLModel.LAST_LOADED_TIME_FIELD
+        + "\": {\"type\": \"date\", \"format\": \"strict_date_time||epoch_millis\"},\n"
+        + "      \""
+        + MLModel.LAST_UNLOADED_TIME_FIELD
+        + "\": {\"type\": \"date\", \"format\": \"strict_date_time||epoch_millis\"},\n"
+        + USER_FIELD_MAPPING
+        + "    }\n"
+        + "}";
 
     public static final String ML_TASK_INDEX_MAPPING = "{\n"
-            + "    \"_meta\": {\"schema_version\": "
-            + ML_TASK_INDEX_SCHEMA_VERSION
-            + "},\n"
-            + "    \"properties\": {\n"
-            + "      \""
-            + MLTask.MODEL_ID_FIELD
-            + "\": {\"type\": \"keyword\"},\n"
-            + "      \""
-            + MLTask.TASK_TYPE_FIELD
-            + "\": {\"type\": \"keyword\"},\n"
-            + "      \""
-            + MLTask.FUNCTION_NAME_FIELD
-            + "\": {\"type\": \"keyword\"},\n"
-            + "      \""
-            + MLTask.STATE_FIELD
-            + "\": {\"type\": \"keyword\"},\n"
-            + "      \""
-            + MLTask.INPUT_TYPE_FIELD
-            + "\": {\"type\": \"keyword\"},\n"
-            + "      \""
-            + MLTask.PROGRESS_FIELD
-            + "\": {\"type\": \"float\"},\n"
-            + "      \""
-            + MLTask.OUTPUT_INDEX_FIELD
-            + "\": {\"type\": \"keyword\"},\n"
-            + "      \""
-            + MLTask.WORKER_NODE_FIELD
-            + "\": {\"type\": \"keyword\"},\n"
-            + "      \""
-            + MLTask.CREATE_TIME_FIELD
-            + "\": {\"type\": \"date\", \"format\": \"strict_date_time||epoch_millis\"},\n"
-            + "      \""
-            + MLTask.LAST_UPDATE_TIME_FIELD
-            + "\": {\"type\": \"date\", \"format\": \"strict_date_time||epoch_millis\"},\n"
-            + "      \""
-            + MLTask.ERROR_FIELD
-            + "\": {\"type\": \"text\"},\n"
-            + "      \""
-            + MLTask.IS_ASYNC_TASK_FIELD
-            + "\" : {\"type\" : \"boolean\"}, \n"
-            + USER_FIELD_MAPPING
-            + "    }\n"
-            + "}";
+        + "    \"_meta\": {\"schema_version\": "
```

Comment:
Reformat this file.

Reply:
it has been the result of reformat. I use spotlessJavaApply task